[NAV-5118] Cleanup readme, contributing and install document

nonifier opened the pull request.

ogeorget-psee committed:
[NAV-5118] Cleanup readme, contributing andd install document

remi-lucas-hove commented on readme.rst:
```diff
@@ -121,7 +83,8 @@ Approach
+| Navitia is instantiated and exposed publicly through api.navitia.io_.
+
 | Developments on Navitia are lead by Hove (previously Kisio Digital and CanalTP).
 | Hove is a subsidiary of Keolis (itself a subsidiary of SNCF, French national railway company).
```
Faudra faire des PR le 1er mars pour changer ça :) !

ogeorget-psee committed:
[NAV-5118] Update install.rst documentation

Replace 'note' with 'warning' to ensure user sees the need to build as
part of a docher container
